cl/api/templates/rest-docs-vlatest.html

```diff
@@ -971,6 +971,15 @@ <h4>Security of RECAP Fetch API</h4>
     <p>If you have questions about our approach, please see <a href="https://free.law/vulnerability-disclosure-policy/">our vulnerability reporting policy and bug bounty program</a>, where you'll find details on contacting us.
     </p>
 
+    <h3 id="alerts-endpoint">{% url "alert-list" version="v3" %}</h3>
+    <p><a href="{% url "alert_help" %}#search-alerts">Search alerts</a> can be made, viewed, modified, and deleted using this endpoint. This can be extremely useful for organizations that need alerts on many topics or on a changing array of topics. To create an alert, send:
+    </p>
+    <ul>
+      <li><strong><code>name</code></strong> &mdash; A human-friendly name for the alert.</li>
+      <li><strong><code>query</code></strong> &mdash; The URL parameters you would get when creating the alert on the front end.</li>
+      <li><strong><code>rate</code></strong> &mdash; How frequently you want your alert query to be checked and alert emails to be sent.</li>
```

Comment:
This needs an example post imo.  Also - a note that some frequencies are only available to some people might be smart?